Lib/shutil.py

```diff
@@ -489,10 +490,13 @@ def _copytree(entries, src, dst, symlinks, ignore, copy_function,
     return dst
 
 def copytree(src, dst, symlinks=False, ignore=None, copy_function=copy2,
-             ignore_dangling_symlinks=False):
+             ignore_dangling_symlinks=False, dirs_exist_ok=False):
     """Recursively copy a directory tree.
 
-    The destination directory must not already exist.
+    The destination directory must not already exist unless `dirs_exist_ok`
+    is true, in which case existing files in the destination tree
+    will be overwritten by matching files from the source tree.
```

Comment:
This docstring should be updated so that it matches the doc.

Reply:
Done in the latest revision.